core/domain/suggestion_services.py

```diff
@@ -593,12 +619,199 @@ def get_translation_suggestions_waiting_longest_for_review_per_lang(
     return [
         get_suggestion_from_model(suggestion_model) for suggestion_model in (
             suggestion_models.GeneralSuggestionModel
-            .get_translation_suggestions_waiting_longest_for_review_per_lang(
+            .get_translation_suggestions_waiting_longest_for_review(
                 language_code)
         )
     ]
 
 
+def _get_plain_text_from_html_content_string(html_content_string):
+    """Retrieves the plain text from the given html content string. RTE element
+    occurrences in the html are replaced by their corresponding name,
+    capitalized in square brackets.
+    eg: <p>Sample1 <oppia-noninteractive-math></oppia-noninteractive-math>
+        Sample2 </p> will give as output: Sample1 [MATH] Sample2.
+    Note: similar logic exists in the frontend in format-rte-preview.filter.ts.
+
+    Args:
+        html_content_string: str. The content html string to convert to plain
+            text.
+
+    Returns:
+        str. The plain text string from the given html content string.
+    """
+
+    def _replace_rte_tag(rte_tag):
+        """Replaces all of the <oppia-noninteractive-**> tags.
+
+        Args:
+            rte_tag: MatchObject. The oppia-noninteractive rte tag.
+
+        Returns:
+            str. The string to replace the rte tag with.
+        """
+        # Convert the MatchObject to a string.
+        rte_tag_string = rte_tag.group(0)
+        # Get the name of the noninteractive tag (ex. math).
+        replace_string = rte_tag_string.split('-')[2].split(' ')[0]
+        if replace_string[-1] == '>':
+            replace_string = replace_string[:-1]
+        replace_string = replace_string.capitalize()
+        replace_string = '%s%s%s' % (' [', replace_string, '] ')
```

Comment:
```suggestion
        replace_string = ' [%s] ' % replace_string
```

It seems like this can be simplified.

Reply:
Good point - thanks!